build.gradle

```diff
@@ -44,7 +44,9 @@ dependencies {
  }
 
 dependencies {
-  implementation platform('com.amazonaws:aws-java-sdk-dynamodb:1.11.732')
+   implementation platform('org.springframework.boot:spring-boot-dependencies:2.1.0.RELEASE')
```

Comment:
Let's not keep spring boot as of now. We will come to that. This commit should only be able to deal with interacting with dynamodb.